daisy/workflow.go

```diff
@@ -83,7 +83,7 @@ func (l *syncedWriter) Flush() error {
 }
 
 func daisyBkt(ctx context.Context, client *storage.Client, project string) (string, error) {
-	dBkt := project + "-daisy-bkt"
+	dBkt := strings.Replace(project, ":", "-", -1) + "-daisy-bkt"
```

Comment:
What is this?

Reply:
Ah, forgot about that. This sanitizes the old style org projects (foo.com:my-project) before creating the daisy bucket.